src/pages/index.js

```diff
@@ -26,26 +29,114 @@ const addCardFormValidator = new FormValidator(
   document.querySelector("#add-card-form")
 );
 
+const profileUserInfo = new UserInfo(".profile__title", ".profile__subtitle");
+
+//Create & Render Card
+
+function createCard(data) {
+  const card = new Card(
```

Comment:
Needs correction: I can't open the avatar edit form and I. can't edit the avatar